draft-gpew-priv-ppm.md

```diff
@@ -862,21 +859,17 @@ body consisting of the following structure:
 
 ~~~
 struct {
-  HpkeConfigId collector_hpke_config_id;
-  opaque enc<1..2^16-1>;
-  opaque payload<1..2^16>;
-} EncryptedAggregateShare;
+  HpkeCiphertext encrypted_output_share;
+} AggregateShareResp;
 ~~~
 
-* `collector_hpke_config_id` is `collector_config.id` from the task parameters
-  corresponding to `CollectReq.task_id`.
-* `enc` is the encapsulated HPKE context, used by the collector to decrypt the
-  aggregate share.
-* `payload` is an encrypted `AggregateShare`.
-
-The leader uses the helper's aggregate share response to respond to the
-collector's collect request (see {{pa-collect}}).
+The leader uses the helper's AggregateShareResp to respond to the collector's
+collect request (see {{pa-collect}}).
 
+After issuing an aggregate-share request for a given batch interval, it is an
+error for the leader to issue any more aggreagte requests for reports in the
+batch interval. These reports will be rejected by helpers as described in
+{{aggregate-request}}.
```

Comment:
It's an error for the leader to issue more than `max_batch_lifetime` requests for a given batch interval, right? As written this states the leader can issue exactly one request.

Reply:
Duplicate of https://github.com/abetterinternet/ppm-specification/pull/174#discussion_r791074329?